recipe/meta.yaml

```diff
@@ -14,42 +14,42 @@ source:
 
 build:
   skip: true  # [win]
+  skip: true  # [py<38]
```

Comment:
please clarify why `[py<38]`, I can't find anything upstream. Note that skipping certain python versions should be specified only if there's some requirement upstream. It shouldn't reflect _anaconda's_ policy about which python versions we build for.